lib/cluster.js

```diff
@@ -57,11 +57,11 @@ function Worker(options) {
 }
 util.inherits(Worker, EventEmitter);
 
-Worker.prototype.kill = function() {
+Worker.prototype.kill = function kill() {
   this.destroy.apply(this, arguments);
 };
 
-Worker.prototype.send = function() {
+Worker.prototype.send = function send() {
```

Comment:
well it looks like it is going in the direction you mentioned , so i have changed it accordingly , 
added arrow function notation for brevity and aesthetic reasons